fmriprep/workflows/bold/base.py

```diff
@@ -817,7 +825,7 @@ def init_func_preproc_wf(
 
         if 'MNI152NLin2009cAsym' in std_spaces:
             carpetplot_wf = init_carpetplot_wf(
-                standard_spaces=std_spaces,
+                standard_spaces=volume_std_spaces,
```

Comment:
I believe this line needs to go away, and pass the list of spaces through a connection to the collated list of templates.
```suggestion
```
That means adding a line right after:
https://github.com/poldracklab/fmriprep/pull/1932/files#diff-e705526667f071ca0c3b04c2bd96272bR834

Like
```Python
                    ('joint_std2anat_xfm', 'inputnode.std2anat_xfm'),
                    ('joint_template', 'inputnode.templates')]),
```

Unfortunately, the carpetplot workflow would not support this directly:

- we need to remove the function argument
- we need to add a new inputnode field
- we need to then connect the new field to `select_std.inputs.keys`